_posts/2019-06-12-cockpit-196.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Cockpit 196
+author: mvo
+date: 2019-06-12
+tags: cockpit linux
+slug: cockpit-196
+category: release
+summary: Machines improvements
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We
+release regularly. Here are the release notes from version 196.
+
+### Machines: Virtual machines can be installed directly from a URL
```

Comment:
URLs already worked for Fedora/RHEL package trees, the new thing is that it now also works for *.iso. The text below clarifies this, but it's a bit ambiguous here.

Reply:
I have changed the title.